src/components/data-management/ProjectMenu.jsx

```diff
@@ -0,0 +1,122 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useSelector, useDispatch } from 'react-redux';
+import {
+  Row, Typography, Space, Button, Col,
+} from 'antd';
+import {
+  updateProject,
+} from '../../redux/actions/projects'; import DownloadData from './DownloadData';
+import fileUploadSpecifications from '../../utils/upload/fileUploadSpecifications';
+import UploadStatus from '../../utils/upload/UploadStatus';
+
+const { Title, Text, Paragraph } = Typography;
+
+const ProjectMenu = (props) => {
+  const {
+    activeProjectUuid, createMetadataColumn, isAddingMetadata,
+    setUploadModalVisible, openAnalysisModal,
+  } = props;
+  const dispatch = useDispatch();
+  const activeProject = useSelector((state) => state.projects[activeProjectUuid]) || false;
+  const projects = useSelector((state) => state.projects);
+  const samples = useSelector((state) => state.samples);
+  const changeDescription = (description) => {
+    dispatch(updateProject(activeProjectUuid, { description }));
+  };
+
+  const checkLaunchAnalysis = () => {
```

Comment:
I would rename this to `canLaunchAnalysis`, to make it obvious that this function returns a boolean that states whether it is ok for the user to launch analysis or not.